__tests__/app.test.js

```diff
@@ -306,5 +306,31 @@ describe("app",()=>{
             })
         })
     })
+    describe("DELETE /api/comments/:comment_id",()=>{
+        test("Status Code: 204 - Correct status code and empty object to be returned when deleting",()=>{
+            return request(app)
+            .delete("/api/comments/1")
+            .expect(204)
+            .then(({body})=>{
+                expect(body).toEqual({})
```

Comment:
That body object is not coming from your API (from Express), but from Supertest. We don't need to check that. In case of a 204 status code, Express would never send any data back.